src/styl/now-value.styl

```diff
@@ -0,0 +1,22 @@
+@import '_variables'
+
+.dollert-now-value-container
+  padding-bottom: 15px
+  text-align: center
+  line-height: 20px
+  background $color-white
```

Comment:
It is better to keep `background-color` inherited. I mean, just remove this line.